test/fixtures/output/kafka/topic/update-success-rest.golden

```diff
@@ -1,5 +1,5 @@
-Updated the following configuration values for topic "topic-exist-rest":
-        Name       | Value | Read-Only  
--------------------+-------+------------
-  compression.type | gzip  | false      
-  retention.ms     |     1 | false      
+Updated the following configuration values for topic "topic-exist-rest" (read-only configs were not updated):
+        Name       |  Value   | Read-Only  
+-------------------+----------+------------
+  compression.type | producer | true       
```

Comment:
Should this say "gzip" or something similar?

Reply:
the tests are updating it to `gzip`. this config is read-only so the backend should return something different to show that it was not updated

Reply:
it actually is! 🔢